### DISCUSSION TITLE
changing Domain name and Certifcate for SunbirdED --> what are the service to be re-deploy

### DISCUSSION THREAD

**Discussion by tsprasath** [Sunbird-Ed/Community#316] [Installation] [answered]
Hi Team,

CSP:GCP
Branch-5.1.0

We are changing the domain name and certification what are the  service need to be re-deployed.

**Comment by beepdot** [comment]
Hi @tsprasath Do you have this info already or not yet? Please let me know and I can provide the info.

**Reply by tsprasath** [reply]
@keshavprasadms I need the info

**Reply by tsprasath** [reply]
@keshavprasadms  Please let me know what services need to deploy if we are going to change the domain name.

**Reply by beepdot** [reply]
- nginx-public-ingress
- Player
- Report
- Notification
- LMS
- Learner
- KnowledgeMw
- Groups
- Dial
- Content
- Cert
- Analytics
- Adminutils
- Keycloak Realm Client Changes - Manual
- Learning
- Yarn (Stop all yarn jobs first, remove the `extract` folder under hduser account, then redeploy)
- All flinks jobs from Knowledge Platform and Datapipeline
- CoreDataProducts
- EdDataProducts
- ml-core-service
- ml-survey-service

Note: This may not be an exhaustive list.

**Comment by chitranshu-keshav** [comment, marked as answer]
@tsprasath If your query is answered, can you please mark this thread as answered ?
